spec/services/set_decline_by_default_spec.rb

```diff
@@ -154,63 +154,6 @@ def expect_all_relevant_decline_by_default_at_values_to_be(expected)
       end
     end
 
-    context 'when the service is run multiple times' do
-      let(:last_decision_at) { 2.business_days.before(now).end_of_day }
-      let(:old_dbd_date) { 8.business_days.after(now).end_of_day }
```

Comment:
ah I was literally pushing a fix for this:
```
let(:old_dbd_date) { 10.business_days.after(last_decision_at) }
```
This way we never depend on `now` to calculate the future business days which messes up on weekends and means 3 days are stuck on the same old dbd date.
Tested with the next 100 days to make sure and no failures.

Reply:
I've pushed https://github.com/DFE-Digital/apply-for-teacher-training/pull/4804 unblock master, we can discuss when your back later today